src/src/openthread-core-efr32-config.h

```diff
@@ -295,4 +285,8 @@
  */
 #define OPENTHREAD_CONFIG_PSA_ITS_NVM_OFFSET 0x20000
 
+#ifndef OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE
```

Comment:
Who sets this? Can we add a header / comment to this define to describe it?

Reply:
This will default to 1 in the config file, once tested and PSA is enabled.

Reply:
Okay, maybe we can add a comment above it to explain what this option is.